.github/workflows/push-dev-environment.yaml

```diff
@@ -40,6 +40,7 @@ jobs:
           REACT_APP_UAL_APP_NAME: 'my-vote-eos-proxy'
           REACT_APP_UAL_API_PROTOCOL: 'https'
           REACT_APP_UAL_API_HOST: 'api.jungle3.alohaeos.com'
+          REACT_APP_PRODUCERS_INFO_API_URL: 'https://graphql-eos.antelope.tools/api/rest/get-producers-info'
```

Comment:
@AngeloCG97 
can we set this to use the jungle API? the jungle API should be deployed now so we can test